### DISCUSSION TITLE
How to add

### DISCUSSION THREAD

**Discussion by Awsipcom** [gicamm/homeassistant-comelit#23] [Q&A]
Hallo, Can you help me add two Comelit devices to Home Assistant? 

host: VEDO IP ADDRESS( where can I find )
    port: 80
    password: VEDO PASSWORD ( where can I find )
    scan_interval: 30

**Comment by Awsipcom** [comment]
host: HUB IP ADDRES 

Do you mean my IP? 

Thanks for the great work

**Reply by gicamm** [reply]
I mean the IP address assigned to the comelit hub
